features/page-breaks.yml.dist

```diff
@@ -2,11 +2,17 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: false
+  baseline: high
+  baseline_low_date: 2021-09-07
+  baseline_high_date: 2024-03-07
   support:
-    chrome: "50"
-    chrome_android: "50"
-    edge: "12"
+    chrome: ≤80
+    chrome_android: "80"
+    edge: ≤80
+    firefox: "92"
+    firefox_android: "92"
+    safari: ≤13.1
+    safari_ios: ≤13.4
```

Comment:
Unfortunately, this is going to be a little messy to align to caniuse's `css-page-break`. See https://github.com/web-platform-dx/web-features/pull/331#discussion_r1580845902 for background on how this pair of features came to be.

What needs to happen now: pick a `compute_from` that aligns us to caniuse. It's so messy, so I won't expect version-perfect results here, but we need to report that Chrome and Edge support this, while Safari and Firefox don't. I _think_ this means pinning to `avoid` values, but I'm not 100% sure.

Reply:
This also provides more context: https://github.com/Fyrd/caniuse/issues/7045

Reply:
Updated to compute from `css.properties.page-break-after.avoid` which appears to be the closest match

Reply:
OK, I've opened https://github.com/web-platform-dx/web-features/issues/1878 to fix this for good.